src/components/organisms/data-table/license-files-status.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import { useTranslations } from 'next-intl';
+import { PlusCircle } from 'lucide-react';
+
+import { Button } from '@/components/ui/button';
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuTrigger,
+} from '@/components/ui/dropdown-menu';
+
+import type { DataTableStatusComponentType } from './types';
+
+const DataTableStatus: DataTableStatusComponentType = ({ filters }) => {
+  const t = useTranslations('Dashboard.Tables');
+  if (filters.get.licenseFileStatus === undefined) return null;
+
+  const handleCheckChange = (isChecked: boolean, item: string) => {
+    if (isChecked) filters.set.licenseFileStatus(item);
+    else filters.helpers.deleteLicenseFilesStatus(item);
+  };
+
+  const items = [
+    { status: t('Status.active'), value: 'Active' },
+    { status: t('Status.rejected'), value: 'Rejected' },
+    { status: t('Status.finished'), value: 'Finished' },
+    { status: t('Status.undeposited'), value: 'Undeposited' },
+  ];
+  return (
+    <DropdownMenu>
```

Comment:
Please check also if the table has `licenseFileStatus` filter allowed in `filtersAllowed`
```suggestion
const DataTableStatus: DataTableStatusComponentType = ({ filters }) => {
  const t = useTranslations('Dashboard.Tables');

  const handleCheckChange = (isChecked: boolean, item: string) => {
    if (isChecked) filters.set.licenseFileStatus(item);
    else filters.helpers.deleteLicenseFilesStatus(item);
  };

  const items = [
    { status: t('Status.active'), value: 'Active' },
    { status: t('Status.rejected'), value: 'Rejected' },
    { status: t('Status.finished'), value: 'Finished' },
    { status: t('Status.undeposited'), value: 'Undeposited' },
  ];
  
  if (filters.get.licenseFileStatus === undefined) return null;
  
  return (
    <DropdownMenu>
```